en/docs/tutorials/create-your-first-rest-api.md

```diff
@@ -41,7 +43,7 @@ Let's create a new REST API component as follows:
 
 2. Click the **REST API** card.
 
-3. Click **Create an API from scratch**.
+3. Click **Start from scratch**.
 
 4. Enter a unique name and a description for the API. For this tutorial, let's enter the following values:
```

Comment:
There is a new field **Access mode** introduced here. Would be good to mention that too right. . Just to say we will go with the default value in this tutorial.

Reply:
This UI changed again to include the options for repository selection in the same page of the dialog box. Updated this step accordingly.